### ISSUE TITLE
question of source code

### ISSUE THREAD

**Issue by YIFanH** [SHI-Labs/FcF-Inpainting#18]
HI @ningyu1991 

thanks for your work, I have a question with source code in training/networks.py line 246.   mask is img[:, -1].unsqueeze(1) 
in  training/training_loop.py img is img=torch.cat([0.5 - mask, erased_img], dim=1) in training/training_loop.py line 262

so  I'm a bit confused about this.

**Comment by praeclarumjj3** [comment]
Hi @YIFanH, thanks for your interest in our work.

Thanks for pointing this inconsistency. Actually, the mask defined in `networks.py L#246` is never used inside the generator (we forgot to clean it after an older experiment). So, it doesn't have any effect on the forward pass of the network. Nonetheless, I have corrected it.

https://github.com/SHI-Labs/FcF-Inpainting/blob/02c781a8b9f695e39524db3490cf926aa37b6a54/training/networks.py#L246

https://github.com/SHI-Labs/FcF-Inpainting/blob/02c781a8b9f695e39524db3490cf926aa37b6a54/training/models.py#L194-L204
